pom.xml

```diff
@@ -86,6 +86,16 @@
     <howlite.version>1.0.0</howlite.version>
   </properties>
 
+  <dependencyManagement>
```

Comment:
if version is set here you can remove it from dependecies in distribution module.